src/components/diagnosticsMessage.ts

```diff
@@ -0,0 +1,56 @@
+export enum DiagnosticsLevel {
+  INFO = 'INFO',
+  WARN = 'WARN',
+  ERROR = 'ERROR'
+}
+
+export abstract class DiagnosticsMessage {
+  protected _type: string;
+  protected _level: DiagnosticsLevel = DiagnosticsLevel.INFO;
+  protected _message: string = '';
+  protected _tip: string = 'No tip available';
+  constructor(type: string) {
+    this._type = type;
+  }
+
+  get message(): string {
+    return this._message;
+  }
+
+  get type(): string {
+    return this._type;
+  }
+
+  get level(): DiagnosticsLevel {
+    return this._level;
+  }
+
+  get tip(): string {
```

Comment:
How is tip supposed to be used?

Reply:
Good Q, missing impl. Added 'Check your cables' tip  to voltage diagnostic message